pkg/apis/provisioning/v0alpha1/types.go

```diff
@@ -50,6 +50,11 @@ type GitHubRepositoryConfig struct {
 	// Whether we should show dashboard previews for pull requests.
 	// By default, this is false (i.e. we will not create previews).
 	GenerateDashboardPreviews bool `json:"generateDashboardPreviews,omitempty"`
+
+	// The prefix for the Grafana data. If specified, Grafana will ignore anything that is outside this prefix.
+	// This is usually something like `grafana/`. Trailing and leading slash are not required.
+	// The path is relative to the root of the repository, regardless of the leading slash.
+	Prefix string `json:"prefix,omitempty"`
```

Comment:
I was talking to Patrick about this one. Wouldn't it be clearer with the name `Path` instead?